### DISCUSSION TITLE
Take (error) colors from the profile of the user

### DISCUSSION THREAD

**Discussion by kingma-sbw** [chocolatey/choco#3368] [Ideas]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

Followup on #2199 
 Why not use the colors set in the users profile? I've mine set in `MicrosoftPowerShell_profile.ps1` thusly:

```
$host.PrivateData.ErrorBackgroundColor="red"
$host.PrivateData.ErrorForegroundColor="white"


# Chocolatey profile
$ChocolateyProfile = "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
if (Test-Path($ChocolateyProfile)) {
  Import-Module "$ChocolateyProfile"
}
```
What would it take to have choco just use the set error colors from the profile?

### Describe The Solution. Why is it needed?

A use might have painstakingly create a PowerShell profile with the proper color for his or her liking. 

### Related Issues

#2199

**Comment by pauby** [comment]
I've converted this to a discussion.

Chocolatey CLI uses its own PowerShell host and doesn't load your user profile.
